treeherder/webapp/api/throttling.py

```diff
@@ -0,0 +1,17 @@
+from rest_framework import throttling
+
+
+class OauthKeyThrottle(throttling.ScopedRateThrottle):
+    def get_cache_key(self, request, view):
```

Comment:
This all looks good, afaict.  But I wasn't able to find an example of where ``get_cache_key`` is called in the Throttle.  I guess I'm just curious: how did you know to do this?  :)